litr-filters/src/main/java/com/linkedin/android/litr/filter/video/gl/ColorMonochromeFilter.java

```diff
@@ -0,0 +1,75 @@
+package com.linkedin.android.litr.filter.video.gl;
+
+import android.graphics.PointF;
+import android.opengl.GLES20;
+
+import androidx.annotation.NonNull;
+
+import java.nio.FloatBuffer;
+
+/**
+ * Created by tapos-datta on 9/14/20.
+ */
+public class ColorMonochromeFilter extends BaseFrameRenderFilter{
+
+    private static final String FRAGMENT_SHADER =
+                    "#extension GL_OES_EGL_image_external : require\n" +
+
+                    "precision mediump float;\n" +
+                    "varying highp vec2 vTextureCoord;\n" +
+                    "uniform lowp samplerExternalOES sTexture;" +
+                    "uniform lowp vec3 newColor;" +
+                    "uniform lowp float intensity;\n" +
+                    "\n"+
```

Comment:
nip: can you match shader styling with other shaders? ("{" on a separate line, no "\n" lines, identation outside of shader text)

Reply:
Matched shader styling with other shaders.